core/src/main/java/tc/oc/pgm/filters/matcher/player/GlidingFilter.java

```diff
@@ -0,0 +1,24 @@
+package tc.oc.pgm.filters.matcher.player;
+
+import static tc.oc.pgm.util.nms.PlayerUtils.PLAYER_UTILS;
+
+import java.util.Collection;
+import java.util.Collections;
+import org.bukkit.event.Event;
+import tc.oc.pgm.api.filter.query.PlayerQuery;
+import tc.oc.pgm.api.player.MatchPlayer;
+import tc.oc.pgm.util.event.PlayerCoarseMoveEvent;
+
+public class GlidingFilter extends ParticipantFilter {
+  public static final GlidingFilter INSTANCE = new GlidingFilter();
```

Comment:
```suggestion
  public static final GlidingFilter INSTANCE = Platform.isModern() ? new GlidingFilter() : StaticFilter.DENY;
```
Probably makes sense to just optimize it away to a deny for legacy